### DISCUSSION TITLE
Does the extension support ForgeUI

### DISCUSSION THREAD

**Discussion by sonny4723** [DominikDoom/a1111-sd-webui-tagcomplete#298] [Q&A] [answered]
I just installed "[a1111-sd-webui-tagcomplete](https://github.com/DominikDoom/a1111-sd-webui-tagcomplete)" into [ForgeUI](https://github.com/lllyasviel/stable-diffusion-webui-forge/). Bit it seems did not work. When I type < , it shows nothing.

**Comment by DominikDoom** [comment, marked as answer]
Yes it does, I just tested it on the most recent version of forge to make sure:
![image](https://github.com/user-attachments/assets/0e13a5ea-6811-459f-9505-120d091955a6)
![image](https://github.com/user-attachments/assets/8395ca8c-4598-4e15-8802-b153b3831616)

*But*: As described in issue #297, due to the recent backend rework of forge, embeddings will not load automatically on startup since the model isn't loaded yet at that time. So if that's what you expected to show after typing <, you'll have to click generate at least once and then click on the extra networks tab refresh button or "Refresh TAC temp files" at the very bottom of the settings page for the extension.
Lora and normal tags will work no problem either way, and for normal tags you don't need the < prefix, just type the word.

EDIT: Forgot to mention, if you don't even get the normal tags to complete, there likely is another issue at play, in that case please tell me if there are any errors that seem related visible in forge's terminal window or in your browser console (CTRL+SHIFT+I)

**Comment by sonny4723** [comment]
@DominikDoom  Thank you so much! I just reinstalled ForgeUI and your extension. Everything works well now! 🙏
